JavaScript/JavaScriptSDK.Tests/CheckinTests/DataLossAnalyzer.tests.ts

```diff
@@ -89,4 +89,5 @@ class DataLossAnalyzerTests extends TestClass {
     }
 }
 
-new DataLossAnalyzerTests().registerTests(); 
\ No newline at end of file
+// Uncomment if you want to use DataLossanalyzer
+// new DataLossAnalyzerTests().registerTests(); 
```

Comment:
Is there a better way to disable DataLossAnalyzer rather then commenting it in 4 places? E.g. some configuration setting to only change one bit back and worth and avoid the case when someone forgets to uncomment/comment one of those.